zappa/wsgi.py

```diff
@@ -109,13 +119,12 @@ def create_wsgi_request(event_info,
             remote_addr = '127.0.0.1'
 
         environ = {
-            'PATH_INFO': get_wsgi_string(path) if should_transform_to_iso_8859_1 else path,
```

Comment:
this might break the decoding of non-ascii query string parameters

Reply:
In my experiment it seems the problem was not here, need more investigation so I reverted it.

I will play around with it after we are able to build a version on it.